src/splinterdb.c

```diff
@@ -17,6 +17,7 @@
 
 #include "clockcache.h"
 #include "splinterdb/splinterdb.h"
```

Comment:
You should be able to delete the `#include "splinterdb/splinterdb.h"` once you make `splinterdb_private.h` self-contained.

Reply:
I know I _should_ be able to, but I explicitly chose to not do it so as I don't think it's the better approach.

Anyway, to reiterate my previous point -- I need this code integrated to /main. And does not seem like I convince you of my reasons.

So, Done.